sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/ServiceBusReceiveLinkProcessor.java

```diff
@@ -129,6 +135,15 @@ public String getLinkName() {
             }));
     }
 
+    /**
+     * Gets the error context associated with this link.
+     *
+     * @return the error context associated with this link.
+     */
+    public AmqpErrorContext getErrorContext() {
```

Comment:
When is this used?

Reply:
I was using earlier, but not now. So removing it.